helm-chart/kuberay-operator/values.yaml

```diff
@@ -134,3 +134,6 @@ env:
 # Enabling this feature contributes to the robustness of Ray clusters.
 # - name: ENABLE_PROBES_INJECTION
 #   value: "true"
+# If set to true, the RayJob CR itself will be deleted if shutdownAfterJobFinishes is set to true. Note that all resources created by the RayJob CR will be deleted, including the K8s Job.
+# - name: DELETE_RAYJOB_CR_AFTER_JOB_FINISHES
+#   value: "true"
```

Comment:
I think the default value is false. Can you (1) add a comment to explain the default value and default behavior (only the RayCluster will be deleted) and (2) change `value: "true"` to `value: "false"` (default value)?

Reply:
Done